scipy/stats/unuran/unuran.pxd

```diff
@@ -0,0 +1,1307 @@
+        UNUR_DISTR_MEXPONENTIAL
+        UNUR_DISTR_COPULA
+        UNUR_DISTR_MCORRELATION
+
+    UNUR_DISTR* unur_distr_beta(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_burr(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_cauchy(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_chi(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_chisquare(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_exponential(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_extremeI(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_extremeII(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_F(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_gamma(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_ghyp(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_gig(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_gig2(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_hyperbolic(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_ig(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_laplace(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_logistic(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_lognormal(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_lomax(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_normal(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_pareto(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_powerexponential(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_rayleigh(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_slash(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_student(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_triangular(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_uniform(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_weibull(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_multinormal(int dim, double* mean, double* covar)
+
+    UNUR_DISTR* unur_distr_multicauchy(int dim, double* mean, double* covar)
+
+    UNUR_DISTR* unur_distr_multistudent(int dim, double nu, double* mean, double* covar)
+
+    UNUR_DISTR* unur_distr_multiexponential(int dim, double* sigma, double* theta)
+
+    UNUR_DISTR* unur_distr_copula(int dim, double* rankcorr)
+
+    UNUR_DISTR* unur_distr_correlation(int n)
+
+    UNUR_DISTR* unur_distr_binomial(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_geometric(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_hypergeometric(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_logarithmic(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_negativebinomial(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_poisson(double* params, int n_params)
+
+    UNUR_DISTR* unur_distr_zipf(double* params, int n_params)
+
+    # FILE* unur_set_stream(FILE* new_stream)
```

Comment:
should we just remove the lines that are commented out? would they be needed for uniform random numbers?

Reply:
> should we just remove the lines that are commented out? would they be needed for uniform random numbers?

I don't think we will be using output streams provided by UNU.RAN (it can be done via Python logging module). So, we can remove these lines.

Reply:
I have removed these now.